program/pimoroni

```diff
@@ -23,7 +23,7 @@ DISCLAIMER
 
 productname="Pimoroni" # the name of the product to install
 prodversion="2016-11-16" # version of the present implementation
-prodbuild="1200" # build for testing purposes
+prodbuild="1400" # build for testing purposes
```

Comment:
prodversion and prodbuild are date and time respectively. optional but STRONGLY advise to reflect ANY changes, as it's a way to identify which implementation is active on a user's system.

... in theory deb fed from get should not fail in updating it, so, again, in theory, it should always be the latest at all time - that is the point of the self-update!

Still it is used in the About box and is an easier way for users to confirm what version of the dashboard they have, as opposed to querying the dpkg database... it could be done in the future though, if it felt that we want to reflect the 'deb version'.